contracts/ToucanRegenBridge.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier:  GPL-3.0
 
-pragma solidity 0.8.14;
+pragma solidity >=0.8.14;
```

Comment:
Why is this change needed?

Reply:
Because I was getting the following error:
`Source file requires different compiler version
(current compiler is 0.8.17+commit.8df45f5f.Emscripten.clang)
- note that nightly builds are considered to be strictly
less than the released version`

Reply:
What process is returning this error? The compiler version should be driven by the [HH config](https://github.com/regen-network/toucan-bridge/blob/7eec01c88e2c26cdda7abc84c2eddc4a61b4bfc0/hardhat.config.js#L46).

Reply:
My IDE was flagging this as an issue and I couldn't benefit from "format on save" and stuff like that, we can delete the commit that introduces this after we're done with the PR if you want

Reply:
We definitely need to revert this change as it has been deliberately pinned to a fixed version so it's clearer to auditors with what version are we deploying.

Reply:
Makes sense to me, I'll revert it after refinement today.

Reply:
Actually, I'll rewrite history to remove it instead of reverting it.